### DISCUSSION TITLE
Reproducibility: Mismatch in the mAP values

### DISCUSSION THREAD

**Discussion by bardia-az** [ultralytics/yolov5#5751] [Q&A] [answered]
I am trying to reproduce the results of the Pretrained Checkpoints table provided in README.md, but my obtained mAP values do not match the ones in the table. I ran the following commands and got the corresponding mAPs:

python val.py --data coco.yaml --img 640 --conf 0.001 --iou 0.65 --weights yolov5m.pt 
mAP@.5=63.3 ; mAP@.5:.95=43.9

python val.py --data coco.yaml --img 640 --conf 0.001 --iou 0.65 --weights yolov5x.pt 
mAP@.5=68.3 ; mAP@.5:.95=49.6

python val.py --data coco.yaml --img 1280 --conf 0.001 --iou 0.65 --weights yolov5x.pt 
mAP@.5=71.8 ; mAP@.5:.95=53.4

Does anybody know why there is this inconsistency?

BTW, I am running the model on Tesla V100-SXM2-32GB.

**Comment by glenn-jocher** [comment, marked as answer]
@bardia-az README reports pycocotools results
![image](https://user-images.githubusercontent.com/26833433/143038777-2bbfaf4d-1132-4658-bab3-14f5a6b3892e.png)
